src/Lean/Util/FoldConsts.lean

```diff
@@ -66,8 +66,26 @@ opaque foldConsts {α : Type} (e : Expr) (init : α) (f : Name → α → α) :
 def getUsedConstants (e : Expr) : Array Name :=
   e.foldConsts #[] fun c cs => cs.push c
 
+/-- Like `Expr.getUsedConstants`, but produce a `NameSet`. -/
+def getUsedConstants' (e : Expr) : NameSet :=
```

Comment:
What about calling it `getUsedConstantsAsSet` so that the name says more than "a bit like that other one"?

Reply:
Done.